app/src/androidTest/java/com/epfl/drawyourpath/mainpage/fragments/ChallengeFragmentTest.kt

```diff
@@ -222,25 +221,27 @@ class ChallengeFragmentTest {
         val scenario = FragmentScenario.launchInContainer(
             ChallengeFragment::class.java,
             Bundle(),
-            R.style.Theme_Bootcamp
+            R.style.Theme_Bootcamp,
         )
 
-        //change value of distance
+        // change value of distance
         onView(withId(R.id.goals_view))
             .perform(
                 actionOnItemAtPosition<RecyclerView.ViewHolder>(
-                    0, replaceTextOnViewChild("", R.id.goal_display_edit_text)
+                    0,
+                    replaceTextOnViewChild("", R.id.goal_display_edit_text),
                 ),
                 actionOnItemAtPosition<RecyclerView.ViewHolder>(
-                    0, pressImeActionButtonOnViewChild(R.id.goal_display_edit_text)
-                )
+                    0,
+                    pressImeActionButtonOnViewChild(R.id.goal_display_edit_text),
+                ),

```

Comment:
Remark: there are these types of changes that I'm not sure are really useful but I guess it is fine to still have them.